package.json

```diff
@@ -66,19 +63,17 @@
     ]
   },
   "dependencies": {
-    "consola": "^2.15.0",
-    "log4js": "^6.3.0",
-    "redact-object": "^3.0.1",
+    "axios": "^0.24.0",
```

Comment:
We are actually using 0.19.x version for a reason - https://github.com/new10com/axios-logger/issues/8 . Can you please revert this change and put axios back to 0.19.x ?

Reply:
Reverted to v^0.19.2